omod/src/main/webapp/pages/changePassword.gsp

```diff
@@ -0,0 +1,128 @@
+<%
+    ui.decorateWith("appui", "standardEmrPage", [ title: ui.message("referenceapplication.home.title") ])
+    ui.includeCss("referenceapplication", "home.css")
+
+    def htmlSafeId = { extension ->
+        "${ extension.id.replace(".", "-") }-${ extension.id.replace(".", "-") }-extension"
+    }
+%>
+
+<div class="row">
+    <div class="col-12 col-sm-12 col-md-12 col-lg-12 homeNotification">
+        ${ ui.includeFragment("coreapps", "administrativenotification/notifications") }
+    </div>
+</div>
+<div class="row">
+    <div class="col-12 col-sm-12 col-md-12 col-lg-12">
+     
+     
+     <fieldset><legend><openmrs:message code="options.login.legend" /></legend>
+<table>
+	<tr>
+		<td><openmrs:message code="options.login.username" /></td>
+		<td>
+			<spring:bind path="opts.username">
+				<input type="text" name="${status.expression}" value="${status.value}" />
```

Comment:
hi @ibacher ,while working on this,i am wondering whether its ok to put jsp code in a gsp page cc @dkayiwa  @mks-d